Annotato/Annotato/ViewModels/Annotation/AnnotationViewModel.swift

```diff
@@ -37,24 +38,45 @@ class AnnotationViewModel: ObservableObject {
     @Published private(set) var isMinimized = true
     @Published private(set) var modelWasUpdated = false
 
+    @Published private(set) var isResolving = false
+    private(set) var resolveBySave = false
+    private(set) var conflictIdx: Int?
+
     init(
         model: Annotation,
         document: DocumentViewModel,
         webSocketManager: WebSocketManager?,
-        palette: AnnotationPaletteViewModel? = nil
+        palette: AnnotationPaletteViewModel? = nil,
+        isResolving: Bool = false,
+        conflictIdx: Int? = nil
     ) {
         self.model = model
         self.document = document
+
+        let mergeConflictsPalette: AnnotationMergeConflictsPaletteViewModel
+        if let conflictIdx = conflictIdx, isResolving {
+            self.conflictIdx = conflictIdx
+            self.isResolving = isResolving
+            mergeConflictsPalette = AnnotationMergeConflictsPaletteViewModel(
+                origin: .zero, width: model.width, height: 50.0, conflictIdx: conflictIdx)
+            self.mergeConflictPalette = mergeConflictsPalette
+        } else {
+            mergeConflictsPalette = AnnotationMergeConflictsPaletteViewModel(
+                origin: .zero, width: 0.0, height: 0.0, conflictIdx: -1)
+            self.mergeConflictPalette = mergeConflictsPalette
+        }
```

Comment:
Instead of having a palette with 0 height and width, is it possible to make it `nil` instead? This piece of code looks too hacky to me

Reply:
Ok done